frontend/src/pages/voting/index.tsx

```diff
@@ -130,108 +132,127 @@ export const VotingPage = () => {
                 <FormattedMessage id="voting.heading" />
               </Heading>
 
-              <Text my={4}>
+              <Text mt={4}>
                 <FormattedMessage id="voting.introduction" />
               </Text>
+              <Link path="/voting-info" variant="arrow-button" sx={{ my: 4 }}>
+                <FormattedMessage id="global.learnMore" />
+              </Link>
             </Box>
 
-            {!isVotingClosed && (
-              <Grid
-                sx={{
-                  gridTemplateColumns: [null, null, "1fr 1fr"],
-                  gridTemplateRows: [
-                    "repeat(4, 46px)",
-                    null,
-                    "repeat(2, 46px)",
-                  ],
-                  mb: 4,
-                }}
-              >
-                <Select
-                  {...select("topic")}
-                  sx={{
-                    background: "orange",
-                    borderRadius: 0,
-                  }}
-                >
-                  <FormattedMessage id="voting.allTopics">
-                    {(text) => <option value="">{text}</option>}
-                  </FormattedMessage>
-                  {data?.conference.topics.map((topic) => (
-                    <option key={topic.id} value={topic.id}>
-                      {topic.name}
-                    </option>
-                  ))}
-                </Select>
-
-                <Select
-                  {...select("language")}
-                  sx={{
-                    background: "violet",
-                    borderRadius: 0,
-                  }}
-                >
-                  <FormattedMessage id="voting.allLanguages">
-                    {(text) => <option value="">{text}</option>}
-                  </FormattedMessage>
-                  {data?.conference.languages.map((language) => (
-                    <option key={language.id} value={language.code}>
-                      {language.name}
-                    </option>
-                  ))}
-                </Select>
-
-                <Select
-                  {...select("vote")}
+            {isVotingClosed !== undefined &&
+              !isVotingClosed &&
```

Comment:
Yes because when there is an error (user cannot vote) isVotingClosed is undefined anyway (but the voting is open actually... ) so setting it as default Close was not right...